content/09.evolution.md

```diff
@@ -72,6 +72,33 @@ This pattern of convergent evolution at some sites could indicate that certain m
 While it is evident that SARS-CoV-2 exhibits moderate potential for ongoing and future evolution, the relationship between mutations and pathogenicity is not yet known.
 Additional data is needed in order to understand patterns of evolutionary change and whether they are likely to affect virulence.
 
+Several factors could promote the evolution of SARS-CoV-2 including immunodeficiency and the transient exposure to antibodies directed against SARS-CoV-2 proteins. 
+A single case study of SARS-CoV-2 infection in an immunocompromised female with chronic lymphocytic leukemia (CLL) and hypogammaglobulinemia [@doi:10.1016/j.cell.2020.10.049] suggested that an accelerated evolution of the virus could occur in conditions of immunodeficiency. 
+A first administration of convalescent plasma did not clear the virus, and an ensuing increase in the genomic diversity in the samples was observed, suggesting an accelerated evolution due to selection pressure. 
+A second administration of convalescent plasma was able to clear the virus from the host 105 days after the initial diagnosis. However, during the whole infection time, the patient was asymptomatic but contagious.
+
+A second single case study in a 45-year old male with antiphospholipid syndrome [@doi:10.1056/NEJMc2031364], confirmed the earlier results of a persistent COVID-19 symptom in immunocompromised patients after 154 days of diagnosis leading to the death of patient. 
+The treatment included Remdesivir and the Regeneron anti-spike protein antibody cocktail. 
+Genomic analyses of the patient nasopharyngeal swabs confirmed an accelerated evolution of the virus through mutations in spike gene and the receptor-binding domain. 
+
+In summary, these two cases studies suggested an accelerated evolution and persistent shedding of the virus in conditions of immunodeficiency. 
+In particular, the first case highlighted the role of convalescent plasma in creating escape variants. 
+In fact, a study [@doi:10.1101/2020.12.28.424451] suggested that exposure to convalescent plasma in vitro could neutralize the virus after 7 days but after 45 days and up to 80 days, viral escape was promoted through mutations in the receptor-binding domain (RBD) and N-terminal domain (NTD). 
+Taken together, these observations could bring crucial information about the conditions and the kinetics of the development of resistance in SARS-CoV-2 and its implications of the design of vaccine regimens or the need for next generation vaccines.
+
+The evolved variants could occur by chance or through an adaptive process that confers a competitive advantage to the virus.
+Variants that had the D614G mutation in the spike glycoprotein seemed to spread faster. 
+However, it has been suggested that the mutation occurred by early chance events rather than by adaptive events [@doi:10.1001/jama.2020.27124].
```

Comment:
```suggestion
However, it has been suggested that the mutation rose in frequency due to early chance events rather than by adaptive events [@doi:10.1001/jama.2020.27124].
```